integration-tests/vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    // Disable watch mode by default - tests will run once and exit
+    watch: false,
```

Comment:
What's the reason for disabling watch mode by default? Were you running into issues? I tend to run these in watch mode.

Reply:
Would using [`vitest run`](https://vitest.dev/guide/cli.html#vitest-run) also work? It's nice to stick to the defaults/conventions of the tool where possible. It's less about enforcing the preferences of an individual (including me) and more about avoiding surprises and fighting against the tool.

Reply:
Unfortunately, `vitest run --watch` doesn't work so I can't use `vitest run` in the mise task without always going into watch mode.

I think the better approach is as I have it now. One really important reason for this is that watch won't rebuild the neon bindings code (in the current setup at least) so its only useful when only modifying the tests.